<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-image: url(:/icons/bg.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <widget class="QListWidget" name="heroList">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>311</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
		background-color: rgba(0, 0, 0, .7);
}

QListWidget::item { 
	border-bottom: 1px solid rgba(255, 255, 255, .3);
	color: white;
}

QListWidget::item:selected {
	background-color: rgba(255, 255, 255, .4);
	color: black;
}</string>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addHeroBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
     </font>
    </property>
    <property name="text">
     <string>Add Selected</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="currentTeam">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>540</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	background-color: rgba(0, 0, 0, .6);
}

QListWidget::item { 
	border-bottom: 1px solid rgba(255, 255, 255, .3);
	color: white;
}

QListWidget::item:selected {
	background-color: rgba(255, 255, 255, .4);
	color: black;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Choose Your Enemies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Current Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeHeroBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
     </font>
    </property>
    <property name="text">
     <string>Remove Selected</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultTableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
	background-color: rgba(0, 0, 0, .6);
	color: white;
}
</string>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="emptyTeamLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Build the enemy team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findCountersBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
     </font>
    </property>
    <property name="text">
     <string>Find Counters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearResultsBtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
     </font>
    </property>
    <property name="text">
     <string>Clear Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="counterResultLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Best Counter Matches</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOverwatch">
    <property name="title">
     <string>Overwatch</string>
    </property>
   </widget>
   <addaction name="menuOverwatch"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
